README

```diff
@@ -0,0 +1,38 @@
+# token-tests
+
+Test suite for a standard "Maker-style" ERC20 token. An example of such a standard token can be found in `src/tests/SampleToken.sol`.
+
+## Getting Started
+
+Run `forge install makerdao/token-tests` in your newly setup repository.
+
+There are two possible ways to use `token-tests`
+
+### 1) Letting `token-tests` handle testing for you
+
+That's the recommended way. Inherit your test contract from `token-tests/TokenTests.sol` instead of `std-forge/Test.sol`. You will then need to set the following inherited variables:
```

Comment:
```suggestion
This is the recommended way. Inherit your test contract from `token-tests/TokenTests.sol` instead of `std-forge/Test.sol`. You will then need to set the following inherited variables:
```